docs/precompiled/0x08.mdx

```diff
@@ -14,10 +14,10 @@ The input must always be a multiple of 6 32-byte values. 0 inputs is valid and r
 |-----------:|-----:|
 | `[0; 31]` (32 bytes) | x1 |
 | `[31; 63]` (32 bytes) | y1 |
-| `[64; 95]` (32 bytes) | x2 |
-| `[64; 95]` (32 bytes) | y2 |
 | `[64; 95]` (32 bytes) | x3 |
+| `[64; 95]` (32 bytes) | x2 |
```

Comment:
I see we have a copy paste error here, could you update the byte ranges?

Reply:
oops! found a few errors on some of the other precompiles as well and pushed the index updates to my fork

Reply:
thanks a lot :-)